File: Capítulo_5/Tabela_Hash/votacao.py
# Criar uma hash votaram para registrar as pessoas que já votaram 
votaram = {}

# Função para verficar quando alguém chegar para votar, confere se o nome já está na hash:
def verificar_eleitor(nome):
    if votaram.get(nome): # Se nome votaram.get(nome) encontra na tabela
        print("Mande Embora!")
    else: # Se nome não estiver na tabela
        votaram[nome] = True
        print("Deixe Votar!")

# Exemplos de uso:
verificar_eleitor("tom")
verificar_eleitor("mike")
verificar_eleitor("mike")

File: Capítulo_5/Tabela_Hash/votacao.js
// Criar uma hash votaram para registrar as pessoas que já votaram
let votaram = {};

//  Função para verficar quando alguém chegar para votar, confire se o nome já está na hash:
function verificarEleitor(nome) {
    if (votaram[nome]) {
        // Se nome em na tabela votaram
        console.log("Mande Embora!");
    } else { // Se nome não estiver na tabela
        votaram[nome] = true;
        console.log("Deixe Votar!");
    }
}

// Exemplos de uso:
verificarEleitor("tom");
verificarEleitor("mike");
verificarEleitor("mike");
